app/helpers/database.js

```diff
@@ -6,7 +6,18 @@ var mongoose = require('mongoose'),
 	flash = require('connect-flash'),
 	session = require('express-session'),
 	log = require('./logger');
-    MongoDBStore = require('connect-mongodb-session')(session);
+		MongoDBStore = require('connect-mongodb-session')(session);
+		
+notForApi = function (fn) {
```

Comment:
Few small things:

You Should change how you are declaring notForApi. Either adding var/const to the front or changing it to `function notForApi () {}` would be okay.

`fn` is a little unclear. I think changing it to `callback` would add a bit of clarity.

It seems everything in the return function is tabbed twice?